src/v/v8_engine/script.cc

```diff
@@ -13,15 +13,21 @@
 #include "utils/file_io.h"
```

Comment:
please write a bit more description in your commit message

```
Script has timeout for running. It uses executor for compile script and run function.
```

there are some suggestions in CONTRIBUTING.md but basically answer the why. why is this commit being added.

Reply:
done